tests/e2e/dmcluster/dmcluster.go

```diff
@@ -192,9 +173,6 @@ var _ = ginkgo.Describe("DMCluster", func() {
 				fmt.Sprintf("/dmctl --master-addr=127.0.0.1:8261 start-relay -s %s %s", sourceID, workerName))
 			framework.ExpectNoError(err, "failed to enable relay log for DmCluster %q", dcName)
 
-			ginkgo.By("try get dm openapi spec")
-			framework.ExpectNoError(tests.GetDMOpenAPISpec(fw, dc.Namespace, controller.DMMasterMemberName(dcName), tests.DMSingleTask, ""), "failed to try get dm openapi spec")
-
```

Comment:
there is an issue when getting openapi, so remove this check.

![0AUOWks8Tb](https://user-images.githubusercontent.com/26425336/183011861-ce0c703e-9bef-43a1-9172-9fa6af755e80.jpg)